src/server/routes/services/all_question.js

```diff
@@ -0,0 +1,8 @@
+import express from 'express'
+import AllQuestion from '../../handler/all_question'
+import rest from '../../utils/router/helpers/rest'
+
+export default express()
+  .get('/all-question-my', AllQuestion.getAllMyQuestions)
+  .get('/all-question/:id', AllQuestion.getAllQuestionsById)
```

Comment:
Rewrite /all-questions and /all-questions/:id
In first one check if it is company
Add test for this check